$NetBSD$

Do not download files during build, let pkgsrc provide it.

--- ../opencv_contrib-4.11.0/modules/face/CMakeLists.txt.orig	2025-02-07 17:04:23.284140051 +0000
+++ ../opencv_contrib-4.11.0/modules/face/CMakeLists.txt
@@ -8,20 +8,6 @@ ocv_define_module(face opencv_core
 )
 # NOTE: objdetect module is needed for one of the samples
 
-set(__commit_hash "8afa57abc8229d611c4937165d20e2a2d9fc5a12")
-set(__file_hash "7505c44ca4eb54b4ab1e4777cb96ac05")
-ocv_download(
-    FILENAME face_landmark_model.dat
-    HASH ${__file_hash}
-    URL
-      "${OPENCV_FACE_ALIGNMENT_URL}"
-      "$ENV{OPENCV_FACE_ALIGNMENT_URL}"
-      "https://raw.githubusercontent.com/opencv/opencv_3rdparty/${__commit_hash}/"
-    DESTINATION_DIR "${CMAKE_BINARY_DIR}/${OPENCV_TEST_DATA_INSTALL_PATH}/cv/face/"
-    ID "data"
-    RELATIVE_URL
-    STATUS res
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/face_landmark_model.dat
+        DESTINATION ${CMAKE_BINARY_DIR}/${OPENCV_TEST_DATA_INSTALL_PATH}/cv/face/input.txt
 )
-if(NOT res)
-  message(WARNING "Face: Can't get model file for face alignment.")
-endif()
